Publish Scala 2.13 build of 2.0.0-SNAPSHOT(2.0.0-RC1)

Running kamon-prometheus 2.0.0-RC1 scala-2.12 with kamon-core 2.0.0-RC1 scala-2.13 cause an exception.
```
2019-07-04 13:59:47.991 | WARN  | main | k.m.ModuleRegistry | Failed to create instance of module [Prometheus Reporter]
java.lang.NoClassDefFoundError: scala.Serializable
	at java.base/java.lang.ClassLoader.defineClassImpl(Native Method)
	at java.base/java.lang.ClassLoader.defineClassInternal(ClassLoader.java:479)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:448)
	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:174)
	at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:1114)
	at java.base/jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:902)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:808)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:766)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:1052)
	at kamon.prometheus.PrometheusReporter.<init>(PrometheusReporter.scala:45)
	at kamon.prometheus.PrometheusReporter.<init>(PrometheusReporter.scala:42)
	at kamon.prometheus.PrometheusReporter$Factory.create(PrometheusReporter.scala:101)
	at kamon.module.ModuleRegistry.createModule(ModuleRegistry.scala:282)
	at kamon.module.ModuleRegistry.$anonfun$load$3(ModuleRegistry.scala:94)
	at kamon.module.ModuleRegistry$$Lambda$486.0000000069D71C20.apply$mcV$sp(Unknown Source)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
	at scala.Option.fold(Option.scala:264)
	at kamon.module.ModuleRegistry.$anonfun$load$2(ModuleRegistry.scala:97)
	at kamon.module.ModuleRegistry.$anonfun$load$2$adapted(ModuleRegistry.scala:90)
	at kamon.module.ModuleRegistry$$Lambda$485.0000000069D71420.apply(Unknown Source)
	at scala.collection.immutable.List.foreach(List.scala:312)
	at kamon.module.ModuleRegistry.load(ModuleRegistry.scala:90)
	at kamon.ModuleLoading.loadModules(ModuleLoading.scala:66)
	at kamon.ModuleLoading.loadModules$(ModuleLoading.scala:65)
	at kamon.Kamon$.loadModules(Kamon.scala:19)
	at kamon.Init.init(Init.scala:45)
	at kamon.Init.init$(Init.scala:42)
	at kamon.Kamon$.init(Kamon.scala:19)
	...
Caused by: java.lang.ClassNotFoundException: scala.Serializable
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:768)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:1052)
	... 88 common frames omitted
```
Is it possible to publish kamon-prometheus 2.0.0-RC1 build for Scala 2.13

--- a/src/main/scala/kamon/prometheus/PrometheusReporter.scala
+++ b/src/main/scala/kamon/prometheus/PrometheusReporter.scala
@@ -125,9 +125,9 @@ object PrometheusReporter {
         startEmbeddedServer = prometheusConfig.getBoolean("start-embedded-http-server"),
         embeddedServerHostname = prometheusConfig.getString("embedded-server.hostname"),
         embeddedServerPort = prometheusConfig.getInt("embedded-server.port"),
-        defaultBuckets = prometheusConfig.getDoubleList("buckets.default-buckets").asScala,
-        timeBuckets = prometheusConfig.getDoubleList("buckets.time-buckets").asScala,
-        informationBuckets = prometheusConfig.getDoubleList("buckets.information-buckets").asScala,
+        defaultBuckets = prometheusConfig.getDoubleList("buckets.default-buckets").asScala.toSeq,
+        timeBuckets = prometheusConfig.getDoubleList("buckets.time-buckets").asScala.toSeq,
+        informationBuckets = prometheusConfig.getDoubleList("buckets.information-buckets").asScala.toSeq,
         customBuckets = readCustomBuckets(prometheusConfig.getConfig("buckets.custom")),
         includeEnvironmentTags = prometheusConfig.getBoolean("include-environment-tags")
       )
@@ -139,7 +139,7 @@ object PrometheusReporter {
     private def readCustomBuckets(customBuckets: Config): Map[String, Seq[java.lang.Double]] =
       customBuckets
         .topLevelKeys
-        .map(k => (k, customBuckets.getDoubleList(ConfigUtil.quoteString(k)).asScala))
+        .map(k => (k, customBuckets.getDoubleList(ConfigUtil.quoteString(k)).asScala.toSeq))
         .toMap
   }
 }

